src/main/scala/com/github/http4sbin/http/services/HeaderService.scala

```diff
@@ -0,0 +1,15 @@
+package com.github.http4sbin.http.services
+
+import org.http4s.HttpService
+import org.http4s.dsl._
+
+import com.github.http4sbin.http.Http4sHelpers.getHeadersAsJson
+
+object HeaderService {
+
+  val service = HttpService {
+    case req @ GET -> Root / "headers" => {
+      Ok(getHeadersAsJson(req.headers).toString)
```

Comment:
If you have `EncodeJson[Headers]`, it can now just be:

`Ok(req.headers.asJson)`